app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
+
+        //encode it as a zip file
+        var content = zip.encode(this._io, temp_dir);
+        this._io.writeFile(false, instanceName, new Buffer(content, 'base64'));
+
+        //delete the temp dir
+        function rmDirRecursive(io, dir){
```

Comment:
I implemented rmDirRecursive myself. But it was clearly influenced by io/readDirRecursive.js